retroarch.c

```diff
@@ -35590,15 +35620,7 @@ static bool retroarch_parse_input_and_config(
                exit(0);
 
             case RA_OPT_LOG_FILE:
```

Comment:
For tidiness, you can move this to the following block (rather than having a distinct entry):

```c
            case 'h':
#ifdef HAVE_CONFIGFILE
            case 'c':
            case RA_OPT_APPENDCONFIG:
#endif
            case 's':
            case 'S':
               break; /* Handled in the first pass */
```